plugin/code_actions.py

```diff
@@ -75,7 +75,7 @@ def request_code_actions_with_diagnostics(view: sublime.View, diagnostics_by_con
 
     actions_at_location = CodeActionsAtLocation(actions_handler)
 
-    for session in sessions_for_view(view, point):
+    for session in sessions_for_view(view):
```

Comment:
Why this change?

Reply:
Can you give an example where it matters that the sessions are sorted by their feature_selector / scopes?

Reply:
From a higher POV:

session_for_view and sessions_for_view are god-functions. I would like to eventually get rid of them. They won't be necessary once we have moved to the one true ViewEventListener design (if that ever happens). I'll make a discussion issue about my thoughts and future lookahead.

Reply:
I thought that this will exclude sessions that don't match scope at point (so, for example, HTML server running in PHP file where HTML server would only match in HTML blocks). But that's not how it worked so I think I'm fine with this change.